includes/class-mailer.php

```diff
@@ -88,8 +88,26 @@ public static function comment_notification_text( $message, $comment_id ) {
 		$post                  = \get_post( $comment->comment_post_ID );
 		$comment_author_domain = \gethostbyaddr( $comment->comment_author_IP );
 
-		/* translators: 1: Comment type, 2: Post title */
-		$notify_message = \sprintf( html_entity_decode( esc_html__( 'New %1$s on your post &#8220;%2$s&#8221;.', 'activitypub' ) ), \esc_html( $comment_type['singular'] ), \esc_html( $post->post_title ) ) . "\r\n\r\n";
+		// Check if this is a reaction to a post or a comment.
+		if ( 0 === (int) $comment->comment_parent ) {
+			$notify_message = \sprintf(
+				/* translators: 1: Comment type, 2: Post title */
+				\html_entity_decode( esc_html__( 'New %1$s on your post &#8220;%2$s&#8221;.', 'activitypub' ) ),
+				\esc_html( $comment_type['singular'] ),
+				\esc_html( $post->post_title )
+			) . PHP_EOL . PHP_EOL;
+
+		} else {
+			$parent_comment = \get_comment( $comment->comment_parent );
+			$notify_message = \sprintf(
+				/* translators: 1: Comment type, 2: Post title, 3: Parent comment author */
+				\html_entity_decode( esc_html__( 'New %1$s on your post &#8220;%2$s&#8221; in reply to %3$s&#8217;s comment.', 'activitypub' ) ),
+				\esc_html( $comment_type['singular'] ),
+				\esc_html( $post->post_title ),
+				\esc_html( $parent_comment->comment_author )
+			) . PHP_EOL . PHP_EOL;
+		}
```

Comment:
should we simply skip the else mail, because we do not show them yet?

Reply:
They're no longer shown in the frontend but they still get saved and are visible in the Comments list. They'll probably also require moderation (although I've not tested that). I'm not sure if it would be more or less confusing to notify about them, tbh. What do you think?